packages/react-native-codegen/src/parsers/parsers-commons.js

```diff
@@ -432,6 +443,44 @@ function buildSchema(
   );
 }
 
+function buildPropSchema(
+  property: Object,
+  types: TypeDeclarationMap,
+  language: string,
```

Comment:
You can try something like;
```suggestion
  getLanguageSchemaInfo: (...) => /* Type */,
  getLanguageTypeAnnotation: (...) => /* Type */,
```